core/app/models/spree/product.rb

```diff
@@ -168,6 +171,18 @@ class Product < Spree::Base
       after_transition to: :draft, do: :after_draft
     end
 
+    def translatable_fields
+      #TODO I think better solution is to implement get_translatable_fields in mobility gem(?)
+      %w[name description meta_description meta_keywords meta_title]
```

Comment:
If this does not exist in Mobility then something like this avoids duplication on our side:
```rb
TRANSLATABLE_FIELDS = %i[name description meta_description meta_keywords meta_title]

translates *TRANSLATABLE_FIELDS, backend: :table
```